tests/test_simple.py

```diff
@@ -2,12 +2,12 @@
 
 # pylint: disable=line-too-long
 
+import dotstrings
```

Comment:
on it

Reply:
No, I think it worked because I was calling from the root instead of the tests dir.

I reverted and deleted the package from my venv, working fine now.